itou/www/approvals_views/tests/test_prolongation.py

```diff
@@ -271,3 +271,89 @@ def test_prolongation_notification_contains_report_link(self):
         prolongation = self.approval.prolongation_set.first()
 
         assert prolongation.report_file.link in email.body
+
+    def test_check_single_prescriber_organization(self):
+        self.client.force_login(self.siae_user)
+        url = reverse("approvals:declare_prolongation", kwargs={"approval_id": self.approval.pk})
+        self.client.get(url)
+
+        reason = ProlongationReason.SENIOR
+        end_at = Prolongation.get_max_end_at(self.approval.end_at, reason=reason)
+
+        post_data = {
+            "end_at": end_at.strftime(DuetDatePickerWidget.INPUT_DATE_FORMAT),
+            "reason": reason,
+            "email": self.prescriber.email,
+            "contact_email": self.faker.email(),
+            "contact_phone": self.faker.phone_number(),
+            "report_file_path": "prolongation_report/memento-mori.xslx",
+            "uploaded_file_name": "report_file.xlsx",
+            "edit": "1",
+        }
+        response = self.client.post(url, data=post_data)
+
+        self.assertContains(response, self.prescriber_organization)
+        self.assertNotContains(response, "Sélectionnez l'organisation du prescripteur habilité")
+
+    def test_check_multiple_prescriber_organization(self):
+        # Link prescriber to another prescriber organization
+        other_prescriber_organization = PrescriberOrganizationWithMembershipFactory(authorized=True)
+        other_prescriber_organization.members.add(self.prescriber)
+
+        self.client.force_login(self.siae_user)
+        url = reverse("approvals:declare_prolongation", kwargs={"approval_id": self.approval.pk})
+        self.client.get(url)
+
+        reason = ProlongationReason.SENIOR
+        end_at = Prolongation.get_max_end_at(self.approval.end_at, reason=reason)
+
+        post_data = {
+            "end_at": end_at.strftime(DuetDatePickerWidget.INPUT_DATE_FORMAT),
+            "reason": reason,
+            "email": self.prescriber.email,
+            "contact_email": self.faker.email(),
+            "contact_phone": self.faker.phone_number(),
+            "report_file_path": "prolongation_report/memento-mori.xslx",
+            "uploaded_file_name": "report_file.xlsx",
+            "edit": "1",
+        }
+        response = self.client.post(url, data=post_data)
+
+        self.assertContains(response, self.prescriber_organization)
+        self.assertContains(response, other_prescriber_organization)
+        assert "Sélectionnez l&#x27;organisation du prescripteur habilité" in response.content.decode("utf-8")
+
+        # self.assertContains(response, "Sélectionnez l'organisation du prescripteur habilité")
+        # does not work (inside an input tag)
+        # if anybody can explain during review...
```

Comment:
si tu affiches directement le response.content, on se rend compte que le `empty_label` de ce champ est pré-échappé "de force", alros que les autres options ne le sont pas.

 Je ne suis pas allé regarder dans le cdoe source de Django / django-boostrap / select2 / etc  à ce point mais je ne serais pas étonné qu'il y a it juste un pré-échappement de la  valeur de l'option par défaut (qui est donnée donc par `empty_label`  et que du coup tu ne puisses pas lire directement ce que tu cherches.

Sinon ce que tu peux faire c'est de faire une vérification par snapshot sur tout le bloc en question, BeautifulSoup fera un unescape des caractères problématiques.

```
    <p>L'organisation s'affichera ci-dessous, s'il y en a plusieurs, selectionnez-la dans la liste.</p>                                     
                                                                                                 
        <div class="form-group is-invalid"><select name="prescriber_organization" hx-post="/approvals/declare_prolongation/689/check_prescriber_email" hx-target="#check_prescriber_email" hx-select="#check_prescriber_email" class="form-control is-invalid" title="" id="id_prescriber_organization">
  <option value="" selected>Sélectionnez l&#x27;organisation du prescripteur habilité</option>                                                                                                
                                                                                  
  <option value="1795">Bernard Boucher</option>                                                     
                                                              
  <option value="1797">Hortense Langlois-Thierry</option>                                                   
                                                                                                        
</select>    
```

Reply:
Pas bête le snapshot. Merci de ton investigation.

Reply:
Et pour info, sans passer par `django-bootstrap`, avec des champs de formulaire "pur django" le problème est aussi présent.